apps/src/music/views/MiniMusicPlayer.tsx

```diff
@@ -35,15 +37,22 @@ const MiniPlayerView: React.FunctionComponent<MiniPlayerViewProps> = ({
   const [currentProjectId, setCurrentProjectId] = useState<string | undefined>(
     undefined
   );
+  const {userId, userType, signInState} = useAppSelector(
+    state => state.currentUser
+  );
+  const analyticsReporter = useMemo(() => new AnalyticsReporter(), []);
```

Comment:
Should this be `useRef`? Any specific reason to use `useMemo` here?

Reply:
Still learning about when to use different React hooks to help performance so didn't have a specific reason not to use `useRef` - will update.

Reply:
Getting a TS error with `useRef`...
'Property 'startSession' does not exist on type 'MutableRefObject<AnalyticsReporter>'.